transform/views/main.py

```diff
@@ -0,0 +1,176 @@
+
+
+@app.route('/html', methods=['POST'])
+def render_html():
+    response = request.get_json(force=True)
+    template = env.get_template('html.tmpl')
+
+    survey = get_survey(response)
+
+    if not survey:
+        return client_error("HTML:Unsupported survey/instrument id")
+
+    logger.info("HTML:SUCCESS")
+
+    return template.render(response=response, survey=survey)
+
+
+@app.route('/pdf', methods=['POST'])
+def render_pdf():
+    survey_response = request.get_json(force=True)
+
+    survey = get_survey(survey_response)
+
+    if not survey:
+        return client_error("PDF:Unsupported survey/instrument id")
+
+    try:
+        pdf = PDFTransformer(survey, survey_response)
+        rendered_pdf = pdf.render()
+
+    except IOError as e:
+        return client_error("PDF:Could not render pdf buffer: %s" % repr(e))
+
+    response = make_response(rendered_pdf)
+    response.mimetype = 'application/pdf'
+
+    logger.info("PDF:SUCCESS")
+
+    return response
+
+
+@app.route('/images', methods=['POST'])
+def render_images():
+    survey_response = request.get_json(force=True)
+
+    survey = get_survey(survey_response)
+
+    if not survey:
+        return client_error("IMAGES:Unsupported survey/instrument id")
+
+    itransformer = ImageTransformer(logger, survey, survey_response)
+
+    try:
+        path = itransformer.create_pdf(survey, survey_response)
+        images = list(itransformer.create_image_sequence(path))
+        index = itransformer.create_image_index(images)
+        zipfile = itransformer.create_zip(images, index)
+        locn = os.path.dirname(path)
+        itransformer.cleanup(locn)
+    except IOError as e:
+        logger.error("HERE")
+        logger.error(e)
+        return client_error("IMAGES:Could not create zip buffer: %s" % repr(e))
+
+    logger.info("IMAGES:SUCCESS")
+
+    return send_file(zipfile, mimetype='application/zip', add_etags=False)
+
+
+@app.route('/cora', methods=['POST'])
+@app.route('/cora/<sequence_no>', methods=['POST'])
+@app.route('/cora/<sequence_no>/<batch_number>', methods=['POST'])
+def cora_view(sequence_no=1000, batch_number=False):
+    survey_response = request.get_json(force=True)
+
+    if batch_number:
+        batch_number = int(batch_number)
+
+    if sequence_no:
+        sequence_no = int(sequence_no)
+
+    survey = get_survey(survey_response)
+
+    if not survey:
+        return client_error("CORA:Unsupported survey/instrument id")
+
+    # TODO: Implementation of CORATransformer
+    ctransformer = CORATransformer(logger, survey, survey_response, batch_number, sequence_no)
+
+    try:
+        pdf = ctransformer.create_formats()
+        ctransformer.prepare_archive()
+        zipfile = ctransformer.create_zip()
+        locn = os.path.dirname(pdf)
+        ctransformer.cleanup(locn)
+    except IOError as e:
+        return client_error("CORA:Could not create zip buffer: %s" % repr(e))
+    except Exception as e:
+        import sys
```

Comment:
Unless there is a need to do the import here in order for it to work, it would be more clear to do all the imports upfront at top of file.